test/multifile_samples/node_modules_erc20/contracts/Foo.sol.instrumented

```diff
@@ -0,0 +1,30 @@
+/// This file is auto-generated by Scribble and shouldn't be edited directly.
```

Comment:
Why is this file part of the PR? Did I forget it originally, or does the PR just need to be rebased?

Reply:
Removed in https://github.com/ConsenSys/scribble/pull/180/commits/c5e0537b6f1900ecd339ed2173c5000b362e6bd8